core/templates/dev/head/css/oppia.css

```diff
@@ -60,6 +60,12 @@
   100% { opacity: 1; }
 }
 
+@-webkit-viewport{width:device-width}
```

Comment:
This is a nit, but can we use a similar style to the rest of the file where we do a line break after `{`? I.e.:

```
@-moz-viewport, @-ms-viewport, ..., @viewport {
  width: device-width
}
```

or similarly for individual rules, if using commas is not supported for @-rules (I'm not sure)? This will ease CSS linting later on.